tools/reviewer/cas_proxy/ProxyClientTool.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.cas_proxy;
+
+import com.google.protobuf.ByteString;
+import com.google.startupos.tools.reviewer.proxy.AuthProtos;
+import com.google.startupos.tools.reviewer.proxy.CASProxyServiceGrpc;
+import com.google.startupos.tools.reviewer.proxy.CASProxyServiceGrpc.CASProxyServiceBlockingStub;
+import com.google.startupos.tools.reviewer.proxy.CASProxyServiceGrpc.CASProxyServiceStub;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+import java.io.File;
+import java.io.FileInputStream;
+import java.nio.file.Paths;
+import java.util.Iterator;
+import java.util.concurrent.Executors;
+
+public class ProxyClientTool {
+
+  public static void uploadFile(CASProxyServiceStub nonBlockingStub, String fileName)
+      throws Exception {
+    StreamObserver<AuthProtos.FileUploadRequest> req =
+        nonBlockingStub.uploadFile(
+            new StreamObserver<AuthProtos.FileUploadResponse>() {
+              @Override
+              public void onNext(AuthProtos.FileUploadResponse value) {
+                System.err.println(value.toString());
+              }
+
+              @Override
+              public void onError(java.lang.Throwable t) {
```

Comment:
`t-<throwable`

Reply:
@oferb fixed in https://github.com/google/startup-os/pull/541/commits/881572de2fce8e235dd42d5a74e171b80f38636c